includes/class-wc-dintero-hp.php

```diff
@@ -73,7 +73,10 @@ private function init_hooks() {
 		add_action( 'woocommerce_pay_order_after_submit', array( $this, 'init_pay' ), 50);
 
 		add_action( 'woocommerce_cancelled_order', array( $this, 'cancel_order' ) );
```

Comment:
I will add a new task in the backlog for avoiding double hooks for all these other hooks as well.